### PR TITLE
Feature/core 779 refactor nav list

### PR DESCRIPTION
**Pull request by datrinh**
- nav-list refactored
- EvanState introduced, please update it whenever you add something to the store

### CODE REVIEW

File: dapps/digital-twin-detail/src/components/container/DataSetForm.ts [hunk 1]
```diff
@@ -242,13 +242,18 @@ export default class DataSetFormComponent extends mixins(EvanComponent) {
...
 
       // parse input value to numbers to check for correct number format with ajv
       if (type === 'number') {
-        controlValue = parseFloat(controlValue);
+        if (Number.isNaN(parseFloat(controlValue))) {
+          controlValue = false;
+        } else {
+          controlValue = parseFloat(controlValue);
+        }
+        // controlValue = Number.isNaN(parseFloat(controlValue));
```
Comment (Tschuck) [inline dapps/digital-twin-detail/src/components/container/DataSetForm.ts]:
```suggestion
```

File: dapps/digital-twin-detail/src/components/container/List.ts [hunk 2]
```diff
@@ -57,6 +58,7 @@ export default class ContainerListComponent extends mixins(EvanComponent) {
     this.container = this.$store.state.container;
     this.schema = this.container.plugin.template.properties[this.name].dataSchema;
     this.setColumns();
+    console.log(this.container);
```
Comment (Tschuck) [inline dapps/digital-twin-detail/src/components/container/List.ts]:
```suggestion
```

File: dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue [hunk 3]
```diff
@@ -20,106 +20,117 @@
...
-          for="searchInput"
-          @click="isActiveSearch = true"
-        >
-          <i
-            class="mdi mdi-magnify mr-1"
-            style="font-size: 22px"
+      <h1>
+        <i class="mdi mdi-eye-off-outline" />
+      </h1>
+      <span>{{ '_evan.no-permission' | translate }}</span>
```
Comment (Tschuck) [inline dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue]: Translations for `_evan.no-permission` are missing

Comment (datrinh) [reply dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue]: The translation comes from one of Karl's PRs :)

**Pull request merged**